helix-core/helix-core-1.0.5-SNAPSHOT.ivy

```diff
@@ -63,6 +63,7 @@ under the License.
     <dependency org="org.yaml" name="snakeyaml" rev="1.30" conf="compile->compile(default);runtime->runtime(default);default->default"/>
     <dependency org="commons-logging" name="commons-logging-api" rev="1.1" conf="compile->compile(*),master(*);runtime->runtime(*)"/>
     <dependency org="io.dropwizard.metrics" name="metrics-core" rev="4.1.14" conf="compile->compile(default);runtime->runtime(default);default->default"/>
+    <dependency org="org.xerial.snappy" name="snappy-java" rev="1.1.7" conf="compile->compile(default);runtime->runtime(default);default->default"/>
```

Comment:
just curious, do we need this in snapshot as well? I thought just adding this in pom.xml is good enough?
Adding it to snapshot would affect backward compatibility or not?

Reply:
My understanding is to keep pom file and ivy file in sync, pom is for maven, and ivy is for gradle. But they should be same. I'll look more into this.